doc/user-guide/jupyter/ipyvtk_plotting.rst

```diff
@@ -51,7 +45,7 @@ If you're using an Anaconda environment, installation is the quite straightforwa
 
     conda env update --name base --file environment.yml
     conda run conda install -y nodejs
-    conda run jupyter labextension install @jupyter-widgets/jupyterlab-manager@2.0 itkwidgets@0.32.0 ipycanvas@0.6.1 ipyevents@1.8.1
+    conda run jupyter labextension install @jupyter-widgets/jupyterlab-manager@2.0 ipycanvas@0.6.1 ipyevents@1.8.1
```

Comment:
to be on the safe side, I would also pin the conda/pypi packages of ipycanvas and ipyevents at the same version.

Reply:
I went the other direction and un-pinned these. That way user's are always installing the latest versions

Reply:
These were pinned during the transition to JupyterLab 3 where a `ipyvtklink`'s dependencies weren't ready yet. But now everything should be working well

Reply:
Actually, I removed these `jupyter labextension *` commands because they aren't needed anymore with JupyterLab 3

Reply:
ok, sounds good to me.